nucypher/blockchain/eth/actors.py

```diff
@@ -27,7 +25,7 @@ class NucypherTokenActor:
     class ActorError(Exception):
         pass
 
-    def __init__(self, checksum_address: str=None, token_agent: NucypherTokenAgent=None):
+    def __init__(self, checksum_address: str = None, token_agent: NucypherTokenAgent = None):
         """
         :param checksum_address:  If not passed, we assume this is an unknown actor
 
```

Comment:
I don't understand what's the deal with `self.checksum_public_address` attribute below (L37). Isn't it always going to raise an AttributeError since it doesn't exist yet?

Reply:
It's a compromise for the case (which occurs in practice, almost without exception) where this class is mixed in with Character.